src/eml2/PropertyKind.h

```diff
@@ -44,6 +44,13 @@ namespace EML2_NS
 		 */
 		DLL_IMPORT_OR_EXPORT std::string getNamingSystem() const;
 
+		/**
```

Comment:
There is not a single allowed uom for a property kind but several.
One of them is called the "base for conversion" regarding Energistic Unit of Measure definition.
I think this is the one we want to return.

From official Energistic Unit of Measure documentation:
A quantity class represents a **set of units** with the same dimension and same underlying measurement 
concept. For example, length is a quantity class. Each quantity class specifies: 
• a **base for conversion** and, optionally, an alternative base. Each base must be unique within the set of 
classes. It is the base that provides a distinction between classes that might otherwise seem similar. 
• a dimension. 
• a set of member units. 
The quantity class can be used to constrain items in a data model because the class defines all of the units 
that are allowed to be used with something that represents a specialization of that class. 
For example, tubular goods like drill pipe are described by their mass per length, typically pounds per foot. 
We would like to have a list of expected units for this kind of thing. So the standard has a quantity class 
called massPerLengthQuantity, which has a property called massPerLengthUom, which is limited to a certain 
list of values, including “lbm/ft”. 
In XML, a type can be created to represent a particular class with a unit attribute that constrains the values to 
the list of symbols that are members of that class, as in the example below. 
In a relational database, triggers can be used to constrain the units that are populated based on the class 
that is assigned to a column. In programming languages, a proxy generator would turn this into an 
enumeration. Other implementations are possible, but XML is the official definition of the standard.

Reply:
Not sure how I should handle your comment. I renamed get UomAsString() to getBaseUomForConversionAsString().

Reply:
I think the method name was fine (the new one as well)
My point was more about the documentation of the method.

Reply:
I updated the documentation as well to reflect the fact that we get the base unit of measure for conversion. Should I draw back the new method name since it can be a breaking change on client side?

Reply:
I would go for "getBaseUomAsString"

Reply:
Done.